homeassistant/components/synology_dsm/binary_sensor.py

```diff
@@ -63,7 +68,7 @@ class SynoDSMSecurityBinarySensor(SynologyDSMBaseEntity, BinarySensorEntity):
     @property
     def is_on(self) -> bool:
         """Return the state."""
-        return getattr(self._api.security, self.entity_type) != "safe"
+        return bool(getattr(self._api.security, self.entity_type) != "safe")
```

Comment:
Is this `bool()` needed? `!=` should pretty clearly return `bool`, doesn't mypy understand that?

Reply:
without `bool()` mypy throws `binary_sensor.py:71: error: Returning Any from function declared to return "bool"  [no-any-return]` since missing type hints on underlying lib

Reply:
Let's add # type: ignore here